tests/python/common/test_heterograph-apply-edges.py

```diff
@@ -260,6 +279,21 @@ def _print_error(a, b):
             _test(lhs, rhs, binary_op)
 
 
+@unittest.skipIf(
+    dgl.backend.backend_name != "pytorch", reason="Only support PyTorch for now"
+)
+@parametrize_idtype
+def test_unibipartite_heterograph_apply_edges(idtype):
```

Comment:
the original bug mentions that it is related to heterogeneous graphs with same source and destination node, should we have a test case for this corner cases?

Reply:
Yes, my function create_random_hetero create a heterograph with same source node and same dest node.

Reply:
I will try to modify the bug description to make it more precise.

Reply:
if create_random_hetero has something special in it, then it should be called out in the method name.
e.g. create_random_hetero -> create_random_hetero_include_same_source_node.

Reply:
OK, I will add comment here.